### ISSUE TITLE
Why is "wsu:" namespace hardcoded? Any standard which enforces this namespace usage?

### ISSUE THREAD

**Issue by nikolaybobrovskiy** [difi/Oxalis-AS4#87]
https://github.com/difi/Oxalis-AS4/blob/b6965860835ab19ad4c365aabe0c5ec6850b4777/src/main/java/no/difi/oxalis/as4/util/SOAPHeaderParser.java#L82

Because of this, for example, this content is failed to be parsed correctly:
```
p4:Id="cert-efbaa4b8-5ead-405e-a086-54a18db30750" xmlns:p4="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"
```

**Comment by FrodeBjerkholt** [comment]
Good question - I will look into it.

**Comment by FrodeBjerkholt** [comment]
I guess that a better implementation would be something like this: 
```java
if (bstElem.getAttributeNS("http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd", "Id").equals(refUri)) {
````

**Comment by FrodeBjerkholt** [comment]
I will fix it in the next release - Most likely this Thursday.

**Comment by FrodeBjerkholt** [comment]
I have a fix ready, but I have to wait with the release for tomorrow.

**Comment by FrodeBjerkholt** [comment]
Fixed in 4.1.5
